src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/ExtractFeatures.java

```diff
@@ -65,6 +66,13 @@
         optional = true)
     protected double hqGenotypeABThreshold = 0.2;
 
+    @Argument(
+        fullName = "query-labels",
+        doc = "Key-value pairs to be added to the extraction BQ query. Ex: --query-labels label1=value1 --query-labels label2=value2",
```

Comment:
would it be clearer to have `Ex: --query-labels key1=value1 --query-labels key2=value2` ? or if that's not right, how are the keys defined?

Reply:
I agree with this comment, but is that what's above (or was this already addressed?)